src/applications/debt-letters/components/DebtLettersList.jsx

```diff
@@ -118,23 +119,7 @@ const DebtLettersList = ({ debtLinks, isVBMSError }) => {
       {!isVBMSError &&
         debtLinks.length < 1 && (
           <div className="vads-u-background-color--gray-lightest vads-u-padding--3 vads-u-margin-top--3">
-            <h3 className="vads-u-font-family--serif vads-u-margin-top--0">
-              VA debt collection is on hold due to the coronavirus
-            </h3>
-            <p className="vads-u-font-family--sans vads-u-margin-bottom--0">
-              We’ve taken action to stop collection on newly established Veteran
-              debt and make it easier for Veterans to request extended repayment
-              plans and address other financial needs during this time.
-            </p>
-            <p className="vads-u-font-family--sans vads-u-margin-bottom--0">
-              You won’t receive any debt collection letters in the mail until
-              after December 31, 2020. For the latest information about managing
-              VA debt, visit our{' '}
-              <a href="http://va.gov/coronavirus-veteran-frequently-asked-questions/">
-                coronavirus FAQs
-              </a>
-              {'.'}
-            </p>
+            <CoronaVirusAlert />
```

Comment:
just curious why this was broke out as a separate component? can we reuse this anywhere else?

Reply:
@craigwheeler good catch - its going to be re-used on 2 other pages